sycl/include/sycl/ext/oneapi/experimental/graph.hpp

```diff
@@ -378,17 +372,30 @@ class __SYCL_EXPORT modifiable_command_graph {
   /// added as dependencies.
   void addGraphLeafDependencies(node Node);
 
+  void print_graph(sycl::detail::string_view path, bool verbose = false) const;
```

Comment:
There was discussion about removing `__INTEL_PREVIEW_BREAKING_CHANGES` here https://github.com/intel/llvm/pull/15694#pullrequestreview-2368454382

Has something changes since then? i.e. past automatic pulldown for release so now okay to change ABI

Reply:
This PR preserves backward compatibility. Newly compiled code would go through the `detail::string_view` code  path, but old pre-compiled code would keep referencing the old symbol (with `std::string` parameter) that we still provide in the library.

Reply:
Thanks for confirming